packages/tasks/src/model-libraries.ts

```diff
@@ -565,6 +565,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/SandAI-org/MAGI-1",
 		countDownloads: `path:"ckpt/vae/config.json"`,
 	},
+	"magenta-realtime": {
+		prettyLabel: "magenta-realtime",
```

Comment:
```suggestion
		prettyLabel: "Magenta RT",
```

this is the name that will show-up here. Usually better to use a human-friendly naming (as for repo name)

<img width="707" height="201" alt="Image" src="https://github.com/user-attachments/assets/0e0e51f0-daef-4f20-909c-f2e55d4dd2fe" />